src/test/rgw/test_multi.py

```diff
@@ -398,11 +398,11 @@ def init(parse_args):
             if is_master_zg and is_master:
                 if bootstrap:
                     # create admin user
-                    arg = ['--display-name', '"Zone User"', '--system']
+                    arg = ['--display-name', '"Zone User"', '--system', '--caps', '"roles=*"']
```

Comment:
does this admin user ever call into the role APIs?

Reply:
no, removed the caps from here